src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -58,17 +60,22 @@ export const retrieveBeneficialOwners = async (req: Request, appData: Applicatio
 };
 
 export const retrieveManagingOfficers = async (req: Request, appData: ApplicationData) => {
+  const transactionId = appData.transaction_id;
+  const overseasEntityId = appData.overseas_entity_id;
   const companyOfficers = await getCompanyOfficers(req, appData[EntityNumberKey] as string);
+  const moPrivateData: ManagingOfficersPrivateData = await getManagingOfficerPrivateData(req, transactionId, overseasEntityId) as ManagingOfficersPrivateData;
```

Comment:
What happens if this call fails? Maybe we should wrap this in a try catch and log an error?

Reply:
Also if there's no MO data returned a log message could be useful too